test/Twilio.Test/Http/WebRequestClientTest.cs

```diff
@@ -188,8 +188,10 @@ public void TestMakeRequestAddsHeadersAndUserAgent()
             Assert.AreEqual("Basic " + authString, this._mockRequest.Headers["Authorization"]);
 
             Assert.IsNotNull(this._mockRequest.UserAgent);
-            Regex rgx = new Regex(@"^twilio-csharp/[0-9.]+(-rc\.[0-9]+)?\s\(\w+\s\w+\)\s.NET/3.5$");
-            Assert.IsTrue(rgx.IsMatch(this._mockRequest.UserAgent));
+            Assert.That(

```

Comment:
By using the native regex assertion, the string and regex will be printed to the console for easier debugging when failing.